### ISSUE TITLE
New idea of the mod

### ISSUE THREAD

**Issue by frankemorra** [SalenSH/-Un-Immersive-Textures#2]
Hello! I'm a great fan of Stronghold Crusader game, and 
I also use these your modded skin textures. I often play stronghold crusader, with Stronghold Europe mod(this mod changes textures to stronghold 1), so I just wanted to ask you, is it possible to replace all Arabian units by European units from Stronghold 1? You see, I just wanna have a mod where I could play with European units(crusaders) from stronghold crusader, against European units from stronghold 1. I hope you're gonna answer me. Thank you)

**Comment by SalenSH** [comment]
@frankemorra Yes it would be possible to exchange all skins from arabian Units with european ones. Sadly i cant code so i cant change their stats, but someone will make a tool on that, too. It is a great idea, honestly, and i will try to look into it when i have time. Sadly i cant work for 1-2 months, but after that i will look working on it or atleast try it😄
